remme/shared/forms/transaction.py

```diff
@@ -61,3 +61,27 @@ def validate_reverse(form, field):
 
         if field.data != 'false':
             raise validators.StopValidation('Incorrect reverse identifier.')
+
+
+class ListReceiptsForm(ProtoForm):
+
+    ids = fields.StringField()
+
+    if isinstance(ids, list):
```

Comment:
@Alladin9393, remove this check.

Reply:
@anastasiia-bilova, It's needed because if id type is not a list then FieldList throw error.